app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -47,23 +47,13 @@
     @Column(name = "operation_type")
     public String operation_type;
 
-/*
-    public static List<BgSendQueue> queue() {
-        return new Select()
-                .from(BgSendQueue.class)
-                .where("success = ?", false)
-                .orderBy("_ID asc")
-                .limit(20)
-                .execute();
-    }
-*/
     public static List<BgSendQueue> mongoQueue() {
         return new Select()
                 .from(BgSendQueue.class)
                 .where("mongo_success = ?", false)
                 .where("operation_type = ?", "create")
-                .orderBy("_ID desc")
-                .limit(30)
+                .orderBy("_ID asc")
```

Comment:
These depends on how fast you upload the data.
As I have said, I can upload 300 packets (or a day data in 30 seconds).
So, in the very rare case that someone did not upload his data for 2 months he will have to wait for 30 minutes.
In most normal cases the delay will only be of a few seconds.